regtests/README.md

```diff
@@ -134,13 +211,13 @@ $ docker run --rm \
 Tests rely on Python 3.8 or higher. `pyenv` can be used to install a current version and mapped to the local directory
 by using
 
-```bash
+```shell
 pyenv install 3.8
 pyenv local 3.8
 ```
 
 Once you've done that, you can run `setup.sh` to generate a python virtual environment (installed at `~/polaris-venv`)
-and download all of the test dependencies into it. From here, `run.sh` will be able to execute any pytest present.
```

Comment:
Seems like a spurious change?

Reply:
IntelliJ says it's incorrect: 
<img width="364" alt="image" src="https://github.com/user-attachments/assets/00cf27ec-edf9-4cd6-8ace-505464365cd7" />

Reply:
IntelliJ says it's _less concise_ in your screenshot which is true, but it's not incorrect and it's not exactly the same.

Reply:
Okay, reverting.

Reply:
I wonder if it's locale specific :sweat_smile: